apps/avatax/src/modules/webhooks/validate-webhook-payload.test.ts

```diff
@@ -68,9 +68,9 @@ describe("verifyCalculateTaxesPayload", () => {
 
     const result = verifyCalculateTaxesPayload(payload);
 
-    expect(result._unsafeUnwrapErr()).toBeInstanceOf(TaxIncompleteWebhookPayloadError);
+    expect(result._unsafeUnwrapErr()).toBeInstanceOf(TaxIncompletePayloadErrors.MissingLinesError);
     expect(result._unsafeUnwrapErr()).toMatchInlineSnapshot(
-      `[TaxIncompleteWebhookPayloadError: No lines found in taxBase]`,
+      `[MissingLinesError: No lines found in taxBase]`,
     );
```

Comment:
suggestion: imho we shouldn't use snapshots that include error class names which can change, instead we could something like this if error message is important for us to test:

```ts
expect(result._unsafeUnwrapErr().message).toBe(/No lines found in taxBase/)
```